pipelines/skylab/optimus/Optimus.changelog.md

```diff
@@ -1,3 +1,11 @@
+# 4.2.0
+
+2020-12-04 (Date of Last Commit)
+
+* Added Gene as row attribute of the loom file
+* Added CellID and input_id as column attributes of the loom file
+* Updated the docker in LoomUtils.wdl task to 0.0.5
```

Comment:
Maybe briefly explain what the change in the docker was about?

Reply:
The two points added above is the changes but I will update what script changed